pyanaconda/modules/common/structures/storage.py

```diff
@@ -196,6 +196,9 @@ def attrs(self) -> Dict[Str, Str]:
             hba-id
             path-id
 
+        Attributes for partitions:
+            parted-partition-name
```

Comment:
The `parted` here is the name of the `parted` project which is used by blivet. I don't think it should be part of the Anaconda API, it's something which might be confusing in the future.

The best candidate so far seems to me `partition-label` but in general I guess we need to document it to be able to understand what it is.

Reply:
Renamed.